node/network/availability-distribution/src/requester/mod.rs

```diff
@@ -219,7 +228,7 @@ impl Stream for Requester {
 async fn query_occupied_cores<Context>(
 	ctx: &mut Context,
 	relay_parent: Hash,
-) -> Result<Vec<OccupiedCore>>
+) -> Result<Vec<OccupiedCore>, NonFatalError>
```

Comment:
it's still used here as a result Error because `?` syntax is much more succinct, but higher level functions are not using it this way